package.json

```diff
@@ -4,12 +4,14 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "start": "node index.js",
+    "start": "nodemon index.js",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "express": "^4.18.2"
+    "dotenv": "^16.0.3",
+    "express": "^4.18.2",
+    "nodemon": "^2.0.22"
```

Comment:
Should be a devDependency

Rest looks good to me.

Generally avoid committing package.json

Reply:
I thought about it but had already made a pull request. So I thought I would update it in the next version.

Thanks for the help, I will surely keep it in mind before making a pull request.

Reply:
Hello @KaProDes, I added nodemon to devDependency.